code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -88,7 +88,7 @@
 	else
 		desc = "The challenge is not taking as many as you can, but guessing what it is before you pass out."
 
-/obj/item/reagent_containers/food/drinks/get_glass_icon(datum/reagent/largest_reagent)
+/obj/item/reagent_containers/food/drinks/drinkingglass/get_glass_icon(datum/reagent/largest_reagent)
 	return largest_reagent?.shot_glass_icon_state
```

Comment:
am i missing something whats the point of this when we have 53-54, you're overwriting with literally the exact same thing.

Reply:
Oh you're right. A mistake in a mistake.